packages/playground/src/components/account/account-register/account-register.tsx

```diff
@@ -41,27 +37,12 @@ export class AccountRegister {
   async formSubmitionHandler() {
     const data = this.changeset;
 
-    const httpResponse = await fetch(
-      "http://localhost:9000/api/create-account",
-      {
-        body: JSON.stringify(data),
-        headers: {
-          "Content-Type": "application/json; charset=utf-8"
-        },
-        method: "POST"
-      }
-    );
-
-    const apiResponse = (await httpResponse.json()) as ApiResponse;
+    const apiResponse = await PlaygroundAPIClient.createAccount(data);
 
-    if (!apiResponse.ok) {
-      const error = apiResponse.error as ErrorResponse;
-      alert(`Something went wrong: ${error.status}`);
-      return;
-    }
-
-    const { multisigAddress } = apiResponse.data as CreateAccountResponseData;
-    window.localStorage.setItem("playground:multisig", multisigAddress);
+    window.localStorage.setItem(
```

Comment:
Do we have any place in the codebase where we define explicitly the kinds of data stored in `window.localStorage`? I'm always a fan of having a single place where schemas are defined rather than places scattered across the codebase that set arbitrary key-value pairs.

Reply:
@snario This is the first time we use `localStorage` at all, so we haven't defined anything yet. I also do not know what else we're going to store at the moment, hence the `playground:*` namespace.

Reply:
OK. Perhaps we should just make a note in the code that, if we ever decide to set this value in some other file or read from it in another file, to abstract it to a more general db accessor? Just want to avoid the scenario where some other `.tsx` file starts overwriting the value for some reason and we end up being unsure how it happens

Reply:
@snario Talking with @ebryn we might define a schema to use `localStorage` more like a DB.